common/decisions/hfe_decision_features.txt

```diff
@@ -78,21 +79,65 @@ hfe_decision_features_exodus = {
 	prerequisites = {
 		hfe_tech_features_exodus
 	}
+	potential = {
+		NOT = { owner = { has_origin = hfe_civic_features_exodus } }
+		NOT = { owner = { has_country_flag = chosen_to_exodus } }
+		NOT = { owner = { has_country_flag = was_government_in_exile } }
+		NOT = {	owner = { has_authority = auth_ancient_machine_intelligence } }
+		owner = {
```

Comment:
why marauders?

Reply:
This was a fail safe check because the Great Khan should not be allowed to perform this action.